File: src/main/java/com/sfh/pokeRogueBot/model/browser/gamejson/BattleEntry.java
package com.sfh.pokeRogueBot.model.browser.gamejson;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

@Data
public class BattleEntry {

    @SerializedName("battleType")
    private int battleType;

    @SerializedName("seedOffsetWaveIndex")
    private Integer seedOffsetWaveIndex;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/browser/gamejson/BattleEntry.kt
package com.sfh.pokeRogueBot.model.browser.gamejson

import com.google.gson.annotations.SerializedName

data class BattleEntry(
    @SerializedName("battleType")
    val battleType: Int,

    @SerializedName("seedOffsetWaveIndex")
    val seedOffsetWaveIndex: Int?
)